src/tools/info/ucx_info.c

```diff
@@ -29,6 +29,8 @@ static void usage() {
     printf("  -p              Show UCP context information\n");
     printf("  -w              Show UCP worker information\n");
     printf("  -e              Show UCP endpoint configuration\n");
+    printf("  -m              Show UCP memory map configuration (default memory size is 1MB. Can be set with -l)\n");
```

Comment:
why need 2 params? let's just jave 1 params with specifying memory size
e.g `ucx_info -m 1g`

Reply:
since this is how you requested it :)
guessed that -m would be like -p/-w/-e and then specify the details for the memory info.
i'll remove -l and keep only -m